apps/communications/contacts/js/loader.js

```diff
@@ -43,7 +43,8 @@
     search: ['search-view', SHARED_CONTACTS],
     multiple_select: ['multiple-select-view'],
     overlay: ['loading-overlay', SHARED_UTILS],
-    ice: ['ice-view']
+    ice: ['ice-view'],
+    import_sim_contacts: ['', SHARED_UTILS]
```

Comment:
Wouldn't be more elegant (and better) to use [null, SHARED_UTILS] instead of an empty string? We can avoid iterate through the DOM looking for a node that doesn't exists. Then in Line 78 we can do something like this:

```javascript
if (elementMapping[name][0]) {
  var node = ...
  toLoad.unshift(node);
}
```

WDYT? @borjasalguero @jpruden92